### PR TITLE
feat(llm): support multi-QA test/generate

### PR DESCRIPTION
**Pull request by jasinliu**
**Note:**
pip install required dependencies first (csv/xls support)

### CODE REVIEW

File: hugegraph-llm/.gitignore
Comment (imbajin) [inline hugegraph-llm/.gitignore:1]: image

could enable the add `end-of-line` option in IDE maybe

File: hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py [hunk 1]
```diff
@@ -602,7 +604,88 @@ def toggle_slider(enable):
             outputs=[raw_out, vector_only_out, graph_only_out, graph_vector_out],
         )
 
-        gr.Markdown("""## 3. Others (🚧) """)
+        gr.Markdown("""## 3. Multi Question Test (❔)""")
```
Comment (imbajin) [inline hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py:607]:
```suggestion
        gr.Markdown("""## 3. User Function """)
```

File: hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py [hunk 2]
```diff
@@ -602,7 +604,88 @@ def toggle_slider(enable):
...
+            df.to_excel(answers_path, index=False)
+            return df, answers_path
+
+        with gr.Row():
+            with gr.Column():
+                questions_file = gr.File(file_types=[".xlsx"], label="Questions File", type="binary")
+            with gr.Column():
+                test_template_file = os.path.join(resource_path, "demo", "questions_template.xlsx")
+                gr.File(value=test_template_file, label="Download Template File")
+                answers_btn = gr.Button("Answer Questions", variant="primary")
```
Comment (imbajin) [inline hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py:669]: Should CSV format be supported?

Comment (jasinliu) [reply hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py:669]: > Should CSV format be supported?

Of course

**Review by imbajin — APPROVED** [review]

**Pull request merged**